### PR TITLE
Serena's first reflection

### PR DESCRIPTION
**Pull request by sereprz**
Also in this PR:
- adjusted summary length to 30 words (this controls how many words are shown in the table of content)
- removed Max's test post

Note that I used `<!--more-->` in my post to cut the summary at a convenient point instead of after 30 words. You can also make it longer using the same trick.

### CODE REVIEW

File: content/post/sp-reflection-1.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
++++
+
+We started a summer fellowship!  <!--more-->
+
+The 1RG summer fellowship is the brain child of Xavier, Max and I. We are manifesting it by establishing peer-to-peer mentorship rituals, and accountability systems in order to spend the next few months researching topics that interests us.
+
+There are a few reasons that pull me towards this experiment:
+1. I wanted to spend more time thinking and writing, but often find it hard to prioritize it and let it slide to the bottom of my TODOs
+2. I have experienced academia as a soul crushing machine and found it impossible to pursue research in that context, despite being very drawn to the exploratory aspects of it
+3. My experience at the recurse centre, a self directed retreat for curious programmers, was life changing and what originally drove me to go back to academia.
```
Comment (wxs) [inline content/post/sp-reflection-1.md:14]: For *my* research I definitely want to have a conversation with you about what was life-changing about Recurse and what aspects of it you think I might learn from.

File: content/post/sp-reflection-1.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+authors = ["Serena Peruzzo"]
++++
+
+We started a summer fellowship!  <!--more-->
+
+The 1RG summer fellowship is the brain child of Xavier, Max and I. We are manifesting it by establishing peer-to-peer mentorship rituals, and accountability systems in order to spend the next few months researching topics that interests us.
+
+There are a few reasons that pull me towards this experiment:
+1. I wanted to spend more time thinking and writing, but often find it hard to prioritize it and let it slide to the bottom of my TODOs
+2. I have experienced academia as a soul crushing machine and found it impossible to pursue research in that context, despite being very drawn to the exploratory aspects of it
```
Comment (wxs) [inline content/post/sp-reflection-1.md:13]: This is of course a common experience in academia. Again for my own research I'm curious to know what aspects of it you found soul-crushing. I'm also curious whether there were things in it, rituals, motivational structures, whatever that you *did* like?

File: content/post/sp-reflection-1.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+3. My experience at the recurse centre, a self directed retreat for curious programmers, was life changing and what originally drove me to go back to academia.
+
+My hunch is that a self-directed fellowship will be a much healthier, joyous way to approach the kind of deep thinking and writing that I want to be doing, and that doing it alongside friends, with agreed upon rituals, will make it that much easier and fun!
+
+The focus of my summer is to research is community, and what it means to build one. Some research questions I'm iterating on, in no particular order:
+- _What does it mean to care for our community?_
+- _What are rituals, relational practices, and other technologies that generate trust and connections?_
```
Comment (wxs) [inline content/post/sp-reflection-1.md:20]: I think you should definitely go to Naryan's workshop at the festival on Saturday :) he experiments a lot with this kind of thing and will be trying some things out with the group

**Review by wxs — APPROVED** [review]
Feels funny to "approve" your reflection, but here you go 👍

File: content/post/sp-reflection-1.md (hunk 2 above)
Comment (sereprz) [reply content/post/sp-reflection-1.md:13]: uhm.. I think that most of the crushing came from a fundamental misalignment between what my expectations of what doing research would be like, and what doing research actually was. I had this idea of research in m

**Pull request merged**